index.html

```diff
@@ -89,6 +89,11 @@ <h1 class="popover__title">
     var directory = [
         {% for entry in site.directory %}
 
+        <!-- id -->
+        {% assign parts = entry.id | split: "/" %}
```

Comment:
hmm so, this "id" will be part of the data.yml files?

who needs to take care that those IDs are unique?
who will assign IDs to persons?

Reply:
They’re build from the path. It’s what users name their folders.

Reply:
awesome!